service/lib/dinstaller/storage/proposal.rb

```diff
@@ -120,11 +125,22 @@ def calculate(settings = {})
       def generate_proposal_settings(settings)
         proposal_settings = Y2Storage::ProposalSettings.new_for_current_product
 
+        config_volumes = read_config_volumes
+        proposal_settings.volumes = config_volumes unless config_volumes.empty?
```

Comment:
what means here empty config volumes? That product does not require any partitioning?

Reply:
I added a comment (and a test case) to clarify that.